akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedTestKit.java

```diff
@@ -26,14 +27,26 @@
 public class EventSourcedTestKit<S, E, ES extends EventSourcedEntity<S, E>>
```

Comment:
This PR should be forward ported to java-spi branch. Maybe not even worth adding in `main` and only add it in `java-spi`, which will become the new main.